extensions/amp-render/1.0/amp-render.js

```diff
@@ -135,8 +95,73 @@ export class AmpRender extends BaseElement {
     return super.isLayoutSupported(layout);
   }
 
+  /**
+   * @return {!UrlReplacementPolicy}
+   * @private
+   */
+  getPolicy_() {
+    const src = this.element.getAttribute('src');
+    // Require opt-in for URL variable replacements on CORS fetches triggered
+    // by [src] mutation. @see spec/amp-var-substitutions.md
```

Comment:
Please add an `amp-render` entry to `spec/amp-var-substitutions.md` since we are referencing it here. And make a note to add a `Substitutions` section to this component's markdown file.

Reply:
I added a TODO for both, since I don't want to publish `amp-var-substitutions.md` with broken links.